kapacitor/1.6/alpine/Dockerfile

```diff
@@ -20,9 +20,7 @@ RUN set -ex && \
     gpg --batch --verify kapacitor-${KAPACITOR_VERSION}_linux_amd64.tar.gz.asc kapacitor-${KAPACITOR_VERSION}_linux_amd64.tar.gz && \
     mkdir -p /usr/src && \
     tar -C /usr/src -xzf kapacitor-${KAPACITOR_VERSION}_linux_amd64.tar.gz && \
-    rm -f /usr/src/kapacitor-*/kapacitor.conf && \
-    chmod +x /usr/src/kapacitor-*/* && \
-    cp -a /usr/src/kapacitor-*/* /usr/bin/ && \
+    cp -ar /usr/src/kapacitor-*/* / && \
```

Comment:
Did the structure of the tarball change in the last release? Or has this been broken for awhile?

Reply:
The dockerfiles in this repo were only introduced in 1.6.1, so I suspect the image has been broken since then, but I don't know the history of this image.